mlir/test/Conversion/GPUToLLVMSPV/gpu-to-llvm-spv.mlir

```diff
@@ -377,3 +377,130 @@ gpu.module @shuffles_mismatch {
     return
   }
 }
+
+// -----
+
+gpu.module @kernels {
+// CHECK:           llvm.func spir_funccc @no_kernel() {
```

Comment:
This has odd whitespaces. Is this intentional?

Reply:
MLIR test gen tool does not like `gpu.module` I think. It does weird stuff (also the variable naming). Manually edited.